napari/layers/labels/labels.py

```diff
@@ -1169,7 +1186,12 @@ def get_status(self, position=None, *, world=False):
         msg : string
             String containing a message that can be used as a status update.
         """
-        msg = super().get_status(position, world=world)
+        msg = super().get_status(
```

Comment:
I'm confused about why this works...? Isn't the superclass _ImageBase?

Reply:
Good catch! Kevin has now fixed this, update here 
https://github.com/napari/napari/blob/954e8a68a8059f1e0f8bed52e5519c9e56837233/napari/layers/labels/labels.py#L1218-L1224